osbenchmark/resources/custom-operations.json.j2

```diff
@@ -0,0 +1,22 @@
+    {
+      "operation": {
+        "name": "index-append",
+        "operation-type": "bulk",
+        "bulk-size": {{bulk_size | default(5000)}},
+        "ingest-percentage": {{ingest_percentage | default(100)}}
+      },{% raw %}
+      "clients": {{bulk_indexing_clients | default(8)}}
+    },{% endraw %}
+    {
+      "operation": {
```

Comment:
For custom operations, we need to remove the `default` operation. This should only be included in the default operations file. Users might not want a match_all query included in their workload if they are already providing their own queries.